.github/workflows/update-cspell-json.yml

```diff
@@ -0,0 +1,73 @@
+name: "Update cspell.json"
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "00 01 1 * *"
```

Comment:
Is it possible to check repo is not fork? Otherwise, PR will be created on forks.